data/hl7-uploads/500HL7Messages.txt-1561400237743

```diff
@@ -0,0 +1,5366 @@
+MSH|^~\&|App3|Facility2|App4|Facility6|201906191028||OMP^O09|3926758|D^I|2.2
```

Comment:
I don't think the files in `hl7-uploads` should be stored in git

Reply:
Good point- let me fix that.